web/styles/breakpoints.sass

```diff
@@ -7,3 +7,61 @@ $laptop: 1080px;
 $tablet: 900px;
 $phablet: 760px;
 $mobile: 480px;
+
+//////////////////////////
+//   SCALING CLASSES    //
+//////////////////////////
+
+.scale
+  &-row
+    &-desktop
+      @media screen and (max-width: $desktop)
+        flex-direction: column !important
+      &-rev
+        @media screen and (max-width: $desktop)
+          flex-direction: column-reverse !important
+
+  &-pad
+    &-sides
+      &-desktop
+        @media screen and (max-width: $desktop)
+          padding-right: $spacing-s !important
+          padding-left: $spacing-s !important
+        &-none
```

Comment:
shouldn't this go under `-tablet` since it's a break point for `$tablet`?

Reply:
ye it should
so many styles to keep track of rip

Reply:
this still needs to go under `-tablet`